icepyx/core/read.py

```diff
@@ -229,10 +278,6 @@ class Read:
         The default describes files downloaded directly from NSIDC (subsetted and non-subsetted) for most products (e.g. ATL06).
         The ATL11 filename pattern from NSIDC is: 'ATL{product:2}_{rgt:4}{orbitsegment:2}_{cycles:4}_{version:3}_{revision:2}.h5'.
 
-    catalog : string, default None
-        Full path to an Intake catalog for reading in data.
-        If you still need to create a catalog, leave as default.
-
```

Comment:
Since we're removing the `catalog` parameter, maybe change the Pull Request title to "Remove intake catalog from Read module"? Just to make it clear in the changelog that catalog is no longer used, because some people might not know what `intake` is.